lib/resourcemgr/resourcemgr.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package api
+package resourcemgr
```

Comment:
Does this actually need its own package?  Where is all of this code called from?

From what I can tell, it has a bunch methods declared as public, but only one of them is ever called from outside of this file in calicoctl/commands/util.go.